postgres/tests/test_statements.py

```diff
@@ -414,6 +415,102 @@ def test_statement_samples_collect(
         conn.close()
 
 
+@pytest.mark.parametrize("pg_stat_activity_view", ["pg_stat_activity", "datadog.pg_stat_activity()"])
+@pytest.mark.parametrize(
+    "user,password,dbname,query,arg,expected_out,expected_keys",
+    [
+        (
+            "bob",
+            "bob",
+            "datadog_test",
+            "SELECT city FROM pg_sleep(3), persons WHERE city = %s",
+            "hello",
+            {
+                'datname': 'datadog_test',
+                'usename': 'bob',
+                'state': 'active',
+                'query_signature': "d9193c18a6f372d8",
+            },
+            ["xact_start", "query_start", "pid", "client_port", "client_addr"],
+        ),
+    ],
+)
+def test_activity_samples_collection(
```

Comment:
Nit: maybe call it `activity_snapshot` to avoid confusion with the other "query samples" we have.